packages/mongodb-chatbot-server/src/app.ts

```diff
@@ -68,7 +67,9 @@ export interface AppConfig {
 */
 export const errorHandler: ErrorRequestHandler = (err, req, res, _next) => {
   const reqId = getRequestId(req);
-  const httpStatus = err.status || 500;
+  const isCorsError = err.message.includes("CORS");
```

Comment:
This works though would prefer if we could keep this handler decoupled from specific error types/statuses